osquery/tables/system/darwin/smc_keys.cpp

```diff
@@ -315,5 +316,148 @@ QueryData genSMCKeys(QueryContext &context) {
 
   return results;
 }
+
+QueryData getTemperatures(QueryContext &context) {
+  QueryData results;
+
+  SMCHelper smc;
+  if (!smc.open()) {
+    return {};
+  }
+
+  // Two sources considered:
+  // * http://superuser.com/a/967056
+  // * https://goo.gl/9kv6Ny
+  std::map<std::string, std::string> keyToName = {
```

Comment:
On second thought, maybe this should be a const symbol outside of the function:
```cpp
const std::map<std::string, std::string> kSMCKeyDescriptions
```
Then it can be used to annotate the `smc_keys` table later on. It also helps to not allocate the structure on the stack each time the table is used. Even though it'll be a rare table, having it as a global static wont hurt the memory footprint.